webdriver/newsession/create-1.py

```diff
@@ -0,0 +1,24 @@
+#META: timeout=long
```

Comment:
Could we give this file a slightly better name? Not seeing what I would find in create.py and create-1.py